pkg/sql/ttl/ttljob/ttljob_processor.go

```diff
@@ -240,43 +283,10 @@ func (t *ttlProcessor) runTTLOnQueryBounds(
 
 	ttlSpec := t.ttlSpec
 	details := ttlSpec.RowLevelTTLDetails
-	cutoff := details.Cutoff
-	ttlExpr := ttlSpec.TTLExpr
 	flowCtx := t.FlowCtx
 	serverCfg := flowCtx.Cfg
 	ie := serverCfg.DB.Executor()
 
-	selectBatchSize := ttlSpec.SelectBatchSize
-
-	aostDuration := ttlSpec.AOSTDuration
-	if aostDuration == 0 {
-		// Read AOST in case of mixed 22.2.0/22.2.1+ cluster where the job started on a 22.2.0 node.
```

Comment:
Should this code stick around for 23.2? I saw this comment on the original PR: https://github.com/cockroachdb/cockroach/pull/101119#pullrequestreview-1377773136

Reply:
no, we decided not to support version skipping from 22.2 directly to 23.2: https://cockroachlabs.slack.com/archives/C05B1BPMJLE/p1696908185131109?thread_ts=1696898969.397849&cid=C05B1BPMJLE